src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -4,15 +4,20 @@
   </ImportGroup>
   <PropertyGroup>
     <ProjectGuid>{51BB6014-43F7-4F31-B8D3-E3C37EBEDAF4}</ProjectGuid>
-    <OutputType>Library</OutputType>
+    <OutputType>Library</OutputType> 
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Dynamo.Wpf</RootNamespace>
     <AssemblyName>DynamoCoreWpf</AssemblyName>
     <DocumentationFile>$(OutputPath)\DynamoCoreWpf.XML</DocumentationFile>
     <EnableDefaultCompileItems>False</EnableDefaultCompileItems>
     <EnableDefaultEmbeddedResourceItems>False</EnableDefaultEmbeddedResourceItems>
-    <PreferredToolArchitecture>x64</PreferredToolArchitecture>
+    <PreferredToolArchitecture>x64</PreferredToolArchitecture> 
   </PropertyGroup>
+    <PropertyGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
```

Comment:
Why not just mark this condition as ` '$(TargetFramework)' == 'net6.0-windows' `?

Reply:
@aparajit-pratap I tried it, but it didn't work because at the time the file runs, the target framework is net6.0 and the condition is skipped. However, to use WPF in net 6 it is necessary that the target framework be net6.0-windows, that's why I make that condition. Otherwise Microsoft.WndowsDesktop.App.WPF does not load and the classes containing the WPF elements cannot be accessed
I think it is not very clear so if you know maybe a better way we could try it

Reply:
The way I did this in my initial PR was to add new Property called `UILib` (or something like that) in the cs_sdk.props file.
When this was set to true, the target framework was set to net6.0-windows + wpf + anything else.
So the only change needed in the .csproj files was to add `<UILib>true</UILib>

Reply:
https://github.com/DynamoDS/Dynamo/pull/12716/files#diff-bbde35c3169d8580248bcea0e6c8e5722086cba412a65c737fb4c8656d095c55R49